tools/IceRpc.Slice.Tools/SliceCCSharpTask.cs

```diff
@@ -37,9 +37,21 @@ public class SliceCCSharpTask : ToolTask
     [Required]
     public string WorkingDirectory { get; set; } = "";
 
+    // <summary>If verbose output should be enabled.</summary>
+    [Required]
+    public bool Verbose { get; set; } = false;
```

Comment:
Could not think of a better name for this additional output.

Reply:
The best solution is to not include this property at all. When you run slicec-cs via this Task, you always get "verbose" output.